src/cmd/linuxkit/build.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/moby/tool/src/moby"
+	log "github.com/sirupsen/logrus"
+)
+
+const defaultNameForStdin = "moby"
```

Comment:
nit: do we want to change this?

Reply:
I missed [@rn's comment](https://github.com/linuxkit/linuxkit/pull/2761#discussion_r152032892), I agree that keeping this file identical makes sense for now so no need to change